content/blog/lektor-32-released/contents.lr

```diff
@@ -0,0 +1,59 @@
+title: Lektor 3.2 Released
+---
+author: Andreas Runfalk
+---
+pub_date: 2020-05-11
+---
+summary: What's changed with Lektor 3.2
+---
+body:
+
+#### text-block ####
+text:
+
+It's been a long time in the making and on behalf of the Lektor Team I'm happy to introduce the 3.2 release of Lektor. As some of you may have noticed the development pace of Lektor has slowed down the two years. Those of you who have had open PRs for a long time, sorry for the delay and we'll try to do better going forward!
```

Comment:
Lektor Team -> Lektor Development Team

Reply:
I also like Lektor Team without Development  a bit better. Are you OK with that as well @goanpeca ?